tensorflow/python/framework/func_graph.py

```diff
@@ -384,7 +383,7 @@ def _capture_by_value(
       else:
         op = ops.get_default_graph().create_op(
             op_type, uncaptured_inputs, dtypes, input_types, name, attrs,
-            op_def, compute_shapes, compute_device)
+            op_def, compute_device)
```

Comment:
compute_device should be passed with as keyword argument now since create_op still has compute_shapes argument before it.

Reply:
Fixed.